From: Alexander Larsson <alexl@redhat.com>
Date: Mon, 27 Aug 2018 12:11:35 +0200
Subject: OCI summary generation: Fix generation of summary on 32bit

The "t" variant type is guint64, so we must cast "0" to it when using
it in a varargs.

Closes: #2024
Approved by: alexlarsson

(cherry picked from commit b8d594e3906b8e12e6b10ce40d769537ca3684aa)
---
 common/flatpak-oci-registry.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/flatpak-oci-registry.c b/common/flatpak-oci-registry.c
index 76f9070..95a0d26 100644
--- a/common/flatpak-oci-registry.c
+++ b/common/flatpak-oci-registry.c
@@ -2147,7 +2147,7 @@ flatpak_oci_index_make_summary (GFile        *index,
 
       g_variant_builder_add_value (refs_builder,
                                    g_variant_new ("(s(t@ay@a{sv}))", ref,
-                                                  0,
+                                                  (guint64)0,
                                                   ostree_checksum_to_bytes_v (fake_commit),
                                                   g_variant_builder_end (ref_metadata_builder)));
       g_variant_builder_add (ref_data_builder, "{s(tts)}",
